src/test/java/com/github/underscore/CollectionsTest.java

```diff
@@ -608,13 +608,19 @@ public boolean test(Integer item) {
 */
     @Test
     public void filter() {
-        final List<Integer> result = U.filter(asList(1, 2, 3, 4, 5, 6),
+        final Iterable<Integer> result = U.filter(new ArrayDeque<Integer>(asList(1, 2, 3, 4, 5, 6)),
```

Comment:
\[FindBugs\] INFO: DLS: Dead store to result in com\.github\.underscore\.CollectionsTest\.filter\(\)